io/GltfWriter.hpp

```diff
@@ -79,6 +80,20 @@ class PDAL_DLL GltfWriter : public Writer
     double m_blue;
     double m_alpha;
     bool m_doubleSided;
+    bool m_colorVertices;
 };
 
+
+struct GltfWriter::ViewData
```

Comment:
I had to move it into the header so that GltfWriterTest.cpp would compile. I'm possibly showing how rusty I am at C++ here, there's probably a better way to fix this that I've forgotten.